distribution/lib/Standard/Microsoft/0.0.0-dev/src/SQLServer_Connection.enso

```diff
@@ -1,7 +1,6 @@
 from Standard.Base import all
 import Standard.Base.Metadata.Display
 import Standard.Base.Visualization.Table_Viz_Data.Table_Viz_Data
-import Standard.Base.Visualization.Table_Viz_Data.Table_Viz_Header
```

Comment:
There is a `lintEnso` [CI error](https://github.com/enso-org/enso/actions/runs/18036395517/job/51325274064?pr=14037#step:15:802) to be fixed by 2736f70edf48d3072a3396069e2f476963a0357f